test.mjs

```diff
@@ -0,0 +1,102 @@
+import {resolve, reject} from 'fluture';
+import S from 'sinon';
+import {
+  deepStrictEqual as eq,
+  throws,
+} from 'assert';
+
+import {
+  Typed,
+  Empty,
+  NotFound,
+  Redirect,
+  Serialized,
+  handler,
+} from '.';
+
+function mock() {
+  return {
+    code: S.stub ().returnsThis (),
+    type: S.stub ().returnsThis (),
+    send: S.stub ().returnsThis (),
+    serializer: S.stub ().returnsThis (),
+  };
+}
+
+const C = x => _ => x;
```

Comment:
That's actually `K`. `C` is `f => a => b => f (b) (a)`

Reply:
:confused: